src/test/java/com/opentok/test/OpenTokTest.java

```diff
@@ -237,7 +237,7 @@ public void testTokenDefault() throws
 
         int apiKey = 123456;
         String apiSecret = "1234567890abcdef1234567890abcdef1234567890";
-        OpenTok opentok = new OpenTok(apiKey, apiSecret);
```

Comment:
Be careful, you are making the readability worse by removing the spacing. Try to follow the current style of the project.